frontend/public/components/nav.jsx

```diff
@@ -131,6 +134,33 @@ HrefLink.propTypes = {
   href: PropTypes.string.isRequired,
 };
 
+class MultiResourceHrefLink extends HrefLink {
```

Comment:
Is it not possible to use `startsWith`?

https://github.com/openshift/console/blob/f3d2338c5561084839036e15ccd42c5640928562/frontend/public/components/nav.jsx#L262-L267

https://github.com/openshift/console/blob/f3d2338c5561084839036e15ccd42c5640928562/frontend/public/components/nav.jsx#L274

Reply:
Updated to use startsWith, thanks @spadgett